README.md

```diff
@@ -97,6 +97,11 @@ Description                                                    | Default
 SERVER_PORT                                                    | 9411
 PITCHFORK_VALIDATORS_ACCEPT_NULL_TIMESTAMPS                    | true
 PITCHFORK_VALIDATORS_MAX_TIMESTAMP_DRIFT_SECONDS               | 3600
+PITCHFORK_INGRESS_KAFKA_ENABLED                                | false
```

Comment:
If I'm not mistaken the 3 properties below should have the `KAFKA` suffix after the `INGRESS` based on [this](https://github.com/HotelsDotCom/pitchfork/pull/28/files#diff-256c57f5b357be82840a9b2207f0dcd5L11).

Reply:
Yeap, good catch.
Fixed.